docs/Android/allowed-apps.md

```diff
@@ -0,0 +1,76 @@
+
+## How to Use
+To inject this configuration, use the API `setSDKControlConfig` offered through the `Crittercism` singleton.
+
+For more on this API, visit: [setSDKControlConfig](crittercism.md#setsdkcontrolconfigconfig)
+
+
+```Java
+/**
+* Sets the WS1 UEM SDK Control Config.
+* IntelligenceSDK will parse for the following properties:
+*  - IntelSDKAllowedApps: JSON Array of apps allowed to transmit DEX data to the UEM console. If not available,
+*    IntelligenceSDK will default to allowing all apps to transmit data.
+*  - DEXData: JSON containing the privacy configuration.
+*  Apps / SDK's are expected to fetch the control config JSON string and call this API. In the case the control
+*  config is not available, a null value should be passed for the config parameter.
+*
+* @param config Json String of the control config provided from the WS1 UEM SDK.
+*/
+public static void setSDKControlConfig(String config)
+```
+
+
+### Control Configuration Attributes
+The following is a table of the Attributes that are currently parsed within the control config.
+
+#### Configuration JSON Attributes Table
+|         Flag        |                                       Description                                       |   Value   |                                                   Outcome                                                   |                                      Notes                                     |
+|:-------------------:|:---------------------------------------------------------------------------------------:|:---------:|:-----------------------------------------------------------------------------------------------------------:|:------------------------------------------------------------------------------:|
+| IntelSDKAllowedApps | JSON Array of Application IDs that are allowed to transmit DEX data to the UEM console. |           |                                                                                                             | This value of this key (JSON object) is parsed only if CaptureDEXData is true. |
+|                     |                                                                                         |  MISSING  |                             All apps will transmit DEX data to the UEM console.                             |                                                                                |
+|                     |                                                                                         | AVAILABLE | Only the matching Application IDs within the array will be allowed to transmit DEX data to the UEM console. |                                                                                |
+
+### Example Payload
+
+```JSON
+  // Only the listed Android apps (App 1 and App 2) will be allowed to transmit DEX data.
+
+  "IntelSDKAllowedApps": [
+    "CRITTERCISM_APP1_ID",  // App 1 ID
+    "CRITTERCISM_APP2_ID"  // App 2 ID
```

Comment:
Should the example include realistic App IDs?  The real JSON for `IntelSDKAllowedApps` won't contain strings.
You could use one of the real App IDs for our Test Apps
- e4037f7d48434605babb944efbc637d300555300
- 8f0d92eab86c45c9affb3a8ff3e025b000555300
Or start with those and alter a few characters. (No need to alter the ending of '00555300'. All the App IDs have that.)